diff -up smartmontools-5.43/smartctl.8.in.ccissman smartmontools-5.43/smartctl.8.in
--- smartmontools-5.43/smartctl.8.in.ccissman	2012-09-25 17:51:25.000000000 +0200
+++ smartmontools-5.43/smartctl.8.in	2012-09-25 17:59:16.502276071 +0200
@@ -527,9 +527,16 @@ such as:
 .nf
 \fBsmartctl \-a \-d cciss,0 /dev/cciss/c0d0\fP    (cciss driver under Linux)
 .fi
+This will give the smart information about the first physical disk drive (0)
+connect to the controller at /dev/cciss/c0d0 . (Disk drive numbering is 0
+based)
 .nf
-\fBsmartctl \-a \-d cciss,0 /dev/sg2\fP    (hpsa or hpahcisr drivers under Linux)
+\fBsmartctl \-a \-d cciss,1 /dev/sg2\fP    (hpsa or hpahcisr drivers under Linux)
 .fi
+This will give the SMART information about the second physical disk drive (1)
+connected to the controller at /dev/sg0
+
+To get the controller device node you will need to run \fBlsscsi \-g\fP.
 .\" %ENDIF OS Linux
 .\" %IF OS FreeBSD
 .nf
