diff -up smartmontools-6.2/os_linux.cpp.fdleak smartmontools-6.2/os_linux.cpp
--- smartmontools-6.2/os_linux.cpp.fdleak	2013-07-05 12:40:38.000000000 +0200
+++ smartmontools-6.2/os_linux.cpp	2016-05-02 15:38:59.369700749 +0200
@@ -2703,6 +2703,7 @@ linux_smart_interface::megasas_dcmd_cmd(
   }
 
   int r = ioctl(fd, MEGASAS_IOC_FIRMWARE, &ioc);
+  ::close(fd);
   if (r < 0) {
     return (r);
   }
